app/retweet_graphs_v2/k_days/classifier.py

```diff
@@ -38,25 +47,20 @@
         )
         storage.upload_bot_probabilities_histogram()
 
-        ## UPLOAD SELECTED ROWS TO BIG QUERY
-        #df = clf.bot_probabilities_df
-        #bots_df = df[df["bot_probability"] > 0.5]
-        #records = [{**{"start_date": date_range.start_date}, **record} for record in bots_df.to_dict("records")]
-        #print("UPLOADING", len(records), "BOT SCORES TO BQ...")
-        ##bq_service.upload_daily_bot_probabilities(records)
-        ##> google.api_core.exceptions.BadRequest: 400 POST https://bigquery.googleapis.com/bigquery/v2/projects/.../tables/daily_bot_probabilities/insertAll: too many rows present in the request, limit: 10000 row count: 36092.
-        ##batches = list(split_into_batches(records))
-        ##for batch in batches:
-        ##    bq_service.upload_daily_bot_probabilities(batch)
-        ##    time.sleep(5)
-
-        # CLEAR MEMORY
+        # UPLOAD SELECTED ROWS TO BIG QUERY (IF POSSIBLE, OTHERWISE CAN ADD FROM GCS LATER)
+        try:
```

Comment:
this is really a belt and suspenders solution. hopefully it never errors out. if it doesn't, will remove the try block later.